patterns/1-initial/extensions-for-sustainable-growth.md

```diff
@@ -54,17 +70,19 @@ In order for the extensions model to be successful, there are few architectural
    - Create criteria for when an extension can be ported to the primary repository, such as adoption of the extension by internal products and demand for the feature.
    - Porting process of the extension to the primary repository will follow more stringent code review guidelines as set by library maintainers.
 
+![Project extensions](../../assets/img/extensions-for-sustainable-growth/extensions-for-sustainable-growth.png)
```

Comment:
In the published pattern the text "Project extensions" will be displayed as a caption below the image.
So please put here whatever you want to be shown for that purpose.